File: devs-tracker-server/src/main/java/com/crionuke/devstracker/core/dto/AppLink.java
package com.crionuke.devstracker.core.dto;

public class AppLink {

    private final String title;
    private final String country;

    public AppLink(String title, String country) {
        this.title = title;
        this.country = country;
    }

    public String getTitle() {
        return title;
    }

    public String getCountry() {
        return country;
    }

    @Override
    public String toString() {
        return getClass().getSimpleName() + "(title=" + title + ", " +
                "country=\"" + country + "\")";
    }
}

File: devs_tracker_client/lib/repositories/server_repository/providers/model/app_link.dart
class AppLink {
  final String title;
  final String country;
  final String url;

  AppLink(this.title, this.country, this.url);

  factory AppLink.fromJson(Map<String, dynamic> json) {
    return AppLink(json["title"], json["country"], json["url"]);
  }

  @override
  String toString() {
    return "(title=\"${title}\", country=${country}, url=\"${url}\")";
  }
}
